tests/js/spec/views/performance/transactionSummary.spec.tsx

```diff
@@ -606,7 +606,7 @@ describe('Performance > TransactionSummary', function () {
         screen.getByRole('button', {name: 'Filter Slow Transactions (p95)'})
       );
 
-      userEvent.click(screen.getByRole('button', {name: 'Slow Transactions (p95)'}));
+      userEvent.click(screen.getAllByText('Slow Transactions (p95)')[1]);
```

Comment:
Not going to request a change here (unless you're eager), but I'd like to suggest that we (at some point) add accessible roles/attributes to the `<CompactSelect />` component. Like so: https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-select-only.html

That way we can do this which reads better and will be less prone to breaking in the future:

```js
userEvent.click(screen.getByRole('option', {name: 'Slow Transactions (p95)'}));
```

Reply:
Ah, interesting. Thanks for linking those issues!